festim/exports/derived_quantities/adsorbed_hydrogen.py

```diff
@@ -0,0 +1,39 @@
+from festim import SurfaceQuantity, k_B
+import fenics as f
+
+
+class AdsorbedHydrogen(SurfaceQuantity):
+    """
+    Computes the hydrogen surface concentration on a surface
+
+    Args:
+        surface (int): the surface id
+
+    Attribtutes
+        surface (int): the surface id
+        export_unit (str): the unit of the derived quantity in the export file
+        title (str): the title of the derived quantity
+        show_units (bool): show the units in the title in the derived quantities
+            file
+        function (dolfin.function.function.Function): the solution function of
+            the hydrogen adsorbed field
+
+    """
+
+    def __init__(self, surface) -> None:
+        super().__init__(field="adsorbed", surface=surface)
+
+    @property
+    def export_unit(self):
+        return f"H m-2"
+
+    @property
+    def title(self):
+        quantity_title = f"Concentration of adsorbed H on surface {self.surface}"
+        if self.show_units:
+            return quantity_title + f" ({self.export_unit})"
+        else:
+            return quantity_title
+
+    def compute(self):
+        return f.assemble(self.function * self.ds(self.surface))
```

Comment:
If this class is used for a non-0D surface then the units will not be H/m2. 
This is fine because this functionality uses Real elements anyway, but I would add assert statements in a few places to make sure this is only used for 1D simulations

Reply:
Shall we change this with `self.function(0)`?

Reply:
We could change it to `self.function(x)` but then what is `x`? it can in theory be anything

I think that keeping this implementation is fine! we just have to raise an error if this is used with a non-1D simulation.